spec/components/chart_spec.rb

```diff
@@ -85,6 +85,14 @@ def component_name
     assert_select "h4.gem-c-heading", text: "hello"
   end
 
+  it "can exclude the heading from the DOM" do
```

Comment:
likewise - it "can be rendered without a visible heading"

Reply:
Good point, I've changed the wording here too 👍